environment.yml

```diff
@@ -0,0 +1,25 @@
+name: privileged_residues
```

Comment:
Is the name needed here w/ your `devenv.sh` script? Maybe better to remove if possible.

Reply:
No, it is not required. `devenv.sh` uses `conda env ... -p ./conda` which overrides the name in `environment.yml`. Will fix.